addons/website_event_exhibitor/static/src/js/event_exhibitor_connect.js

```diff
@@ -46,8 +46,6 @@ publicWidget.registry.eventExhibitorConnect = publicWidget.Widget.extend({
 
         if (this.userEventManager) {
             document.location = this.$el.data('sponsorUrl');
-        } else if (!this.eventIsOngoing && !this.isParticipating) {
-            document.location = this.$el.data('registerUrl');
```

Comment:
Would clean the data as well, again very unlikely that people customized that specific attribute.